scripts/app.js

```diff
@@ -1,44 +1,43 @@
 (function(module) {
```

Comment:
Everything else looks up to our best Functional Programming standards. My only comment would be to potentially try implementing something we haven't explicitly done in class, but using the same concepts/methods (I'm tasking myself with the same thing for my portfolio). While I can't see your index.html page here maybe a little more content would spice things up as well, unless you're going for a super minimalistic approach (then disregard this line). See if you can use .reduce()/filter() elsewhere! Maybe a new dataset? Experiment!